src/EA.Weee.Web/Areas/Admin/Controllers/AddAatfController.cs

```diff
@@ -55,7 +69,77 @@ public async Task<ActionResult> SearchResults(SearchResultsViewModel viewModel)
                 return View(viewModel);
```

Comment:
I think it's missing a unit test for returning the view

Reply:
Was calling get method instead of post in one of the tests. Now fixed.